src/v/redpanda/application.cc

```diff
@@ -487,7 +487,7 @@ void application::wire_up_services() {
      * running opertions before shutting down the RPC server, preventing it to
      * wait on background dispatch gate `close` call.
      */
-    _deferred.emplace_back([this] { controller->shutdown_input().get(); });
```

Comment:
delete the comment too? needs rethinking of the comments as well. don't think they apply anaymore.